src/extensions/index.ts

```diff
@@ -8,6 +8,7 @@ export const extensions: {
   'cite-apa': 'cite-apa'
```

Comment:
Wondering if `extensions/index.ts` needs to be under source control, as it's generated? Feel free to tell me it does, I'm just trying to understand the codebase in a bit more detail 😄

Reply:
Agreed. Ignored all generated files in 059983d1a0e11cca1b1fa56cd849efcabbbb4b82. If someeone else could do the git hooks that's be great.